eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -44,14 +45,38 @@ where
             .into_iter()
             .filter(|l| re.is_match(&l.address) && !exceptions.is_match(&l.address));
 
-        for Link { line_start, .. } in links {
+        for Link { address, line_start } in links {
+            let (suggestion, extra_help) = if let Some(caps) = eip_re.captures(&address) {
+                let relevant_part = &caps[2];
+                if relevant_part.starts_with("EIPS/") {
+                    (format!("./{}.md", relevant_part.trim_start_matches("EIPS/")), true)
```

Comment:
Hard coding "EIPS/" won't work for the [ERCs](https://github.com/ethereum/ERCs/) repository.